lsp-ui-sideline.el

```diff
@@ -305,23 +312,33 @@ CURRENT is non-nil when the point is on the symbol."
         (eol (line-end-position)))
     (when (bound-and-true-p flycheck-mode)
       (dolist (e (flycheck-overlay-errors-in bol (1+ eol)))
-        (let* ((message (--> (flycheck-error-format-message-and-id e)
-                             (car (split-string it "\n"))
-                             (replace-regexp-in-string "[\n\t ]+" " " it)))
-               (len (length message))
-               (level (flycheck-error-level e))
-               (face (if (eq level 'info) 'success level))
-               (margin (lsp-ui-sideline--margin-width))
-               (message (progn (add-face-text-property 0 len 'lsp-ui-sideline-global nil message)
-                               (add-face-text-property 0 len face nil message)
-                               message))
-               (string (concat (propertize " " 'display `(space :align-to (- right-fringe ,(lsp-ui-sideline--align len margin))))
-                               message))
-               (pos-ov (lsp-ui-sideline--find-line len t))
-               (ov (and pos-ov (make-overlay pos-ov pos-ov))))
-          (when pos-ov
-            (overlay-put ov 'after-string string)
-            (push ov lsp-ui-sideline--ovs)))))))
+        (let* ((lines (--> (flycheck-error-format-message-and-id e)
+                           (split-string it "\n")))
+               (display-lines (if lsp-ui-sideline-diagnostic-all-lines
+                                lines
+                                (list (car lines)))))
+          (dolist (line display-lines)
+            (let* ((message (string-trim (replace-regexp-in-string "[\t ]+" " " line)))
+                   (len (length message))
+                   (level (flycheck-error-level e))
+                   (face (if (eq level 'info) 'success level))
+                   (margin (lsp-ui-sideline--margin-width))
+                   (message (progn (add-face-text-property 0 len 'lsp-ui-sideline-global nil message)
+                                   (add-face-text-property 0 len face nil message)
+                                   message))
+                   (string (concat (propertize " " 'display `(space :align-to (- right-fringe ,(lsp-ui-sideline--align len margin))))
+                                   message))
+                   (pos-ov (lsp-ui-sideline--find-line len nil))
+                   (ov (and pos-ov (make-overlay pos-ov pos-ov))))
+              (when pos-ov
+                (overlay-put ov 'after-string string)
+                (push ov lsp-ui-sideline--ovs)))
```

Comment:
please place these on one line

Reply:
(I mean the brackets)

Reply:
Is there a reference for the style? I think I corrected this.